src/test/java/org/fundacionjala/coding/kevincristhian/VigenereCryptoTest.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * tests for vigenere.
+ */
+public class VigenereCryptoTest {
+    private static final String LEMON = "LIMON";
+
+    /**
+     * test for encode.
+     */
+    @Test
+    public void encode() {
+        VigenereCrypto vigenereCrypto = new VigenereCrypto();
```

Comment:
You are repeating this line of code in all your tests
Please move it to a @before method.